### PR TITLE
Sentry integration

### PR DESCRIPTION
**Pull request by akolesen**
## Summary by CodeRabbit

- **New Features**
    - Integrated Sentry for enhanced error tracking and diagnostics.
- **Chores**
    - Updated testing dependencies for improved reliability.
- **Documentation**
    - Added Sentry configuration details for source code integration.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -58,6 +71,18 @@ dependencies {
...
 }
 
+tasks.configureEach {
+    if (name == "generateSentryBundleIdJava") {
+       mustRunAfter(tasks.generateMainServiceApolloSources)
+    }
+    if (name == "sentryCollectSourcesJava") {
+       mustRunAfter(tasks.generateMainServiceApolloSources)
+    }
+    if (name == "sentryCollectSourcesJava") {
```
Comment (palina-krukovich) [inline build.gradle:81]: Seems, like these two if's can be merged

File: src/main/java/io/kontur/disasterninja/DisasterNinjaApplication.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package io.kontur.disasterninja;
 
+import io.sentry.Sentry;
```
Comment (palina-krukovich) [inline src/main/java/io/kontur/disasterninja/DisasterNinjaApplication.java:3]: Looks like an unused dependency

Comment (akolesen) [reply src/main/java/io/kontur/disasterninja/DisasterNinjaApplication.java:3]: It's instrumenting the whole app by evaluating this import

Comment (palina-krukovich) [reply src/main/java/io/kontur/disasterninja/DisasterNinjaApplication.java:3]: Cool, ok:)

**Review by palina-krukovich — APPROVED** [review]

**Pull request merged**